text/0111-auto-resource-detection.md

```diff
@@ -0,0 +1,101 @@
+# Automatic Resource Detection
+
+Describes a mechanism to support auto-detection of resource information.
```

Comment:
This should be a one-line summary that describes the changed introduced by this document, not the document itself.

```suggestion
Introduce a mechanism to support auto-detection of resources.
```